src/js/zqd/zqd.js

```diff
@@ -124,6 +124,12 @@ export class ZQD {
       "-zeekrunner",
       zeekRunnerCommand(this.zeekRunner)
     ]
+
+    if (process.platform !== "win32") {
```

Comment:
```suggestion
    // For unix systems, pass posix pipe read file descriptor into zqd process.
    // In the event of Brim getting shutdown via `SIGKILL`, this will let zqd
    // know that it has been orphaned and to shutdown.
    if (process.platform !== "win32") {
```